### DISCUSSION TITLE
Hide a specific string in Table of contents

### DISCUSSION THREAD

**Discussion by joumaico** [squidfunk/mkdocs-material#2500] [Q&A] [answered]
![mkdocs](https://user-images.githubusercontent.com/62525840/112758945-69e80300-9023-11eb-9937-db738c389ebe.jpg)

Do we have an option to hide or better alter the link text in Table of contents rather than being dependent to the title of the heading? As you can see from the image above I need to hide attributes/parameters of the function in the Table of contents. For example, hide `number` from `algorithm.luhn()` in Table of contents while `algorithm.luhn(number)` will remain as heading in the body.

**Comment by squidfunk** [comment]
Converting to discussion, since this is a question.

**Comment by squidfunk** [comment, marked as answer]
You could use [custom labels](https://python-markdown.github.io/extensions/toc/#custom-labels), which are supported by the table of contents extension.

**Reply by joumaico** [reply]
Thank you Martin, I appreciate your work btw. :)
